test/Conversion/XTenNNToTorch/grid_sample.mlir

```diff
@@ -0,0 +1,29 @@
+
+// (c) Copyright 2024 Advanced Micro Devices, Inc. All Rights reserved.
```

Comment:
Is this the same for other tests? There is also that weird blank line that we normally don't have.

Reply:
Yes, copyright seems to be necessary now. I'll remove the blank line and merge the PR